x/ecocredit/simulation/genesis.go

```diff
@@ -291,10 +188,45 @@ func getBatchSequence(ctx context.Context, sStore api.StateStore, projectKey uin
 	return seq.NextSequence, nil
 }
 
-func genGenesisState(ctx context.Context, r *rand.Rand, simState *module.SimulationState, ss api.StateStore, ms marketplaceapi.StateStore) error {
+func genGenesisState(ctx context.Context, simState *module.SimulationState, ss api.StateStore, ms marketplaceapi.StateStore) error {
 	accs := simState.Accounts
+	r := simState.Rand
+
 	metadata := simtypes.RandStringOfLength(r, simtypes.RandIntBetween(r, 5, 100))
 
+	// generate core params
+	if err := ss.AllowListEnabledTable().Save(ctx, &api.AllowListEnabled{
```

Comment:
```suggestion
	if err := ss.AllowListEnabledTable().Insert(ctx, &api.AllowListEnabled{
```

Reply:
We don't have `Insert` method for `singleton` tables.